assets/button.scss

```diff
@@ -1,9 +1,20 @@
-.v-btn--block {
+//.v-btn--block {
+//  height: 50px !important;
+//  background-color: var(--v-primary-base) !important;
+//  transition: 0.4s;
+//  &:hover {
+//    box-shadow: 0px 3px 1px -2px rgba(0, 0, 0, 0.2), 0px 2px 2px 0px rgba(0, 0, 0, 0.14), 0px 1px 5px 0px rgba(0, 0, 0, 0.12),
+//      inset 0 -50px 0 0 var(--v-secondary-base);
+//  }
+//}
+.v-btn--primary{
```

Comment:
Did you check CSS  with XD file CSS part of component ?

Reply:
> Did you check CSS with XD file CSS part of component ?

couldn't find the style in xd file but it's almost accurate.

Reply:
select Export for Web in Share > Development, and publish the design specs. You can view and copy CSS code of your design from the CSS snippets section.